interface/frontend/base.html

```diff
@@ -1,58 +1,61 @@
+  <link href="../static/css/project.css" rel="stylesheet">
+  {% block head %}
+  {% endblock %}
+</head>
 
-  <body>
+<body>
 
-    <nav class="navbar navbar-toggleable-md navbar-inverse bg-inverse fixed-top">
-      <img src="../static/images/c-logo-light.svg"/>
-      <a class="navbar-brand" href="#">Concept To Clinic</a>
-      <div class="navbar-collapse collapse" id="navbar">
-        <ul class="navbar-nav justify-content-center">
-          <li class="nav-item active">
-            <a class="nav-link" href="#">Open image</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#">Detect and Select</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#">Annotate and Segment</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#">Report and Export</a>
-          </li>
-        </ul>
-        <ul class="nav navbar-nav mx-auto justify-content-end">
-          <li class="nav-item">
-            <a class="nav-link">Version: {{ short_hash }}</a>
-          </li>
-        </ul>
-      </div>
-    </nav>
-    <div class="container">
-      {% block content %}
-      {% endblock %}
-    </div><!-- /.container -->
+  <nav class="navbar navbar-toggleable-md navbar-inverse bg-inverse fixed-top">
+    <img src="../static/images/c-logo-light.svg" />
+    <a class="navbar-brand" href="#">Concept To Clinic</a>
+    <div class="navbar-collapse collapse" id="navbar">
+      <ul class="navbar-nav justify-content-center">
+        <li class="nav-item active">
+          <a class="nav-link" href="#">Open image</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">Detect and Select</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">Annotate and Segment</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">Report and Export</a>
+        </li>
+      </ul>
+      <ul class="nav navbar-nav mx-auto justify-content-end">
+        <li class="nav-item">
+          <a class="nav-link">Version: {{ short_hash }}</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <div class="container">
+    {% block content %}
+    {% endblock %}
+  </div>
+  <!-- /.container -->
 
-    <!-- Bootstrap core JavaScript
+  <!-- Bootstrap core JavaScript
     ================================================== -->
-    <!-- Placed at the end of the document so the pages load faster -->
-    <script src="../static/js/jquery-3.1.1.slim.min.js"></script>
-    <script src="../static/js/tether.min.js"></script>
-    <script src="../static/js/bootstrap.min.js"></script>
-    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
-    <script src="../static/js/ie10-viewport-bug-workaround.js"></script>
-  </body>
+  <!-- Placed at the end of the document so the pages load faster -->
+  <script src="../static/js/jquery-3.1.1.slim.min.js"></script>
```

Comment:
Sorry for not noticing this before, but I think we should be using Django's staticfiles framework. This ensures  — or can at least provide -- some kind of cachebusting support.

Reply:
@lamby you're right, thanks for the suggestion. I updated everything accordingly.